src/Common/src/Interop/Windows/mincore/Interop.UnsafeCreateFile.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [System.Security.SecurityCritical]  // auto-generated
+        internal static SafeFileHandle UnsafeCreateFile(
```

Comment:
Transitional- the logic for long paths will probably have to go here as the concept doesn't appear to exist for WinRT. Want to get some basic scenarios up and working and then move away from the other. It is also possible that I might change the signature here as I try implementing.